<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Option</class>
 <widget class="QWidget" name="Option">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>筛选</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_13">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="8,2">
     <property name="spacing">
      <number>40</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9" stretch="3,7">
       <property name="spacing">
        <number>25</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,3">
         <property name="spacing">
          <number>50</number>
         </property>
         <item>
          <widget class="QRadioButton" name="select_all_randomly_button">
           <property name="text">
            <string>随机背诵全部单词</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
           <item>
            <widget class="QLabel" name="info2">
             <property name="text">
              <string>最多:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="word_number_spin">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QRadioButton" name="select_by_condition_button">
           <property name="text">
            <string>按条件筛选</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,9">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>15</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QCheckBox" name="tag_cb">
                 <property name="text">
                  <string>按标签筛选</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9">
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="gridLayout">
                   <property name="spacing">
                    <number>15</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QCheckBox" name="zk_cb">
                     <property name="text">
                      <string>中考</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QCheckBox" name="gk_cb">
                     <property name="text">
                      <string>高考</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QCheckBox" name="cet4_cb">
                     <property name="text">
                      <string>CET4</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QCheckBox" name="cet6_cb">
                     <property name="text">
                      <string>CET6</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QCheckBox" name="ielts_cb">
                     <property name="text">
                      <string>雅思</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QCheckBox" name="toefl_cb">
                     <property name="text">
                      <string>托福</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QCheckBox" name="ky_cb">
                     <property name="text">
                      <string>考研</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="3">
                    <widget class="QCheckBox" name="oxford_cb">
                     <property name="text">
                      <string>牛津核心3000词</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QCheckBox" name="collins_cb">
                 <property name="text">
                  <string>柯林斯星级</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,8,1">
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="gridLayout_2">
                   <property name="horizontalSpacing">
                    <number>15</number>
                   </property>
                   <item row="0" column="3">
                    <widget class="QCheckBox" name="collins4_cb">
                     <property name="text">
                      <string>4</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QCheckBox" name="collins1_cb">
                     <property name="text">
                      <string>1</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QCheckBox" name="collins2_cb">
                     <property name="text">
                      <string>2</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QCheckBox" name="collins3_cb">
                     <property name="text">
                      <string>3</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="4">
                    <widget class="QCheckBox" name="collins5_cb">
                     <property name="text">
                      <string>5</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QCheckBox" name="bnc_cb">
                 <property name="text">
                  <string>英国国家语料库词频顺序</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,9">
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,0,10">
                   <item>
                    <widget class="QLabel" name="info0">
                     <property name="text">
                      <string>前</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="bnc_spin">
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <number>1000</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_9">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Minimum</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QSlider" name="bnc_slider">
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <number>1000</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QCheckBox" name="frq_cb">
                 <property name="text">
                  <string>当代语料库词频顺序</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,9">
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0,0,10">
                   <item>
                    <widget class="QLabel" name="info1">
                     <property name="text">
                      <string>前</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="frq_spin">
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <number>1000</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_10">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Minimum</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QSlider" name="frq_slider">
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <number>1000</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <widget class="QCheckBox" name="pos_cb">
                 <property name="text">
                  <string>按词性筛选</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,9">
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QCheckBox" name="verb_cb">
                     <property name="text">
                      <string>动词</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="n_cn">
                     <property name="text">
                      <string>名词</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="prep_cb">
                     <property name="text">
                      <string>介词</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="adv_cb">
                     <property name="text">
                      <string>副词</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="adj_cb">
                     <property name="text">
                      <string>形容词</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="other_cb">
                     <property name="text">
                      <string>其它</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="finish_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>设定完成</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="return_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>finish_button</sender>
   <signal>clicked()</signal>
   <receiver>Option</receiver>
   <slot>emit_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>797</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>return_button</sender>
   <signal>clicked()</signal>
   <receiver>Option</receiver>
   <slot>return_main()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnc_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bnc_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnc_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bnc_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frq_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>frq_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frq_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>frq_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>emit_settings()</slot>
  <slot>return_main()</slot>
  <slot>is_random()</slot>
  <slot>is_condition()</slot>
 </slots>
</ui>
